assets/custom-theme.scss

```diff
@@ -207,3 +207,23 @@ kbd {
   font-weight: normal;
 }
 
+.green {
+	color: rgb(0, 255, 0);
```

Comment:
```suggestion
	color: green;
```

Reply:
`green` has a different color than `rgb(0, 255, 0);`: it has value `rgb(0, 128, 0);`. 
Do you prefer it?

Reply:
Didn't know. No then it's fine